web/app/components/new/doc-form.ts

```diff
@@ -68,13 +56,10 @@ export default class NewDocFormComponent extends Component<NewDocFormComponentSi
   /**
    * An object containing error messages for each applicable form field.
    */
-  @tracked protected formErrors = { ...FORM_ERRORS };
-
-  /**
-   * Whether the summary is more than 200 characters.
-   * Used in the template to gently discourage long summaries.
-   */
-  @tracked protected summaryIsLong = false;
+  @tracked protected formErrors: DocFormErrors = {
+    title: null,
+    productAbbreviation: null,
+  };
```

Comment:
The initial implementation had the right idea of initially assigning an empty object and then validating against it. However, you'll see below that it was never updated.